### DISCUSSION TITLE
[blocks]: Code Block like you know react-syntax-highlighter but more modern and shadcn looking simple yet nice looking

### DISCUSSION THREAD

**Discussion by fazin-ahamed** [shadcn-ui/ui#4624] [Blocks Request]
### Description

Code Block like you know react-syntax-highlighter but more modern and shadcn looking simple yet nice looking with many language especially auto formatting, i want like a perfect lib that i want to use without any other

### Example

https://react-syntax-highlighter.github.io/react-syntax-highlighter/demo/prism.html

**Comment by ahmedjidar** [comment]
https://vercel.com/geist/code-block
